src/utils.js

```diff
@@ -329,24 +329,92 @@ exports.isEmpty = function(object) {
     return true;
   };
 
-  /**
-   * Iterate object with the function
-   * falls back to es5 `forEach`
-   * @param {Array|Object} object
-   * @param {Function(value, key, object)} fn
-   */
+/**
+ * Iterate object with the function
+ * falls back to es5 `forEach`
+ * @param {Array|Object} object
+ * @param {Function(value, key, object)} fn
+ */
 exports._each = function(object, fn) {
-    if (this.isEmpty(object)) return;
-    if (this.isFn(object.forEach)) return object.forEach(fn);
+  if (this.isEmpty(object)) return;
+  if (this.isFn(object.forEach)) return object.forEach(fn);
 
-    var k = 0,
-        l = object.length;
+  var k = 0,
+      l = object.length;
 
-    if (l > 0) {
-      for (; k < l; k++) fn(object[k], k, object);
-    } else {
-      for (k in object) {
-        if (hasOwnProperty.call(object, k)) fn(object[k], k, object);
+  if (l > 0) {
+    for (; k < l; k++) {
+      if (fn(object[k], k, object) === false) {
+        return;
       }
     }
+  } else {
+    for (k in object) {
+      if (hasOwnProperty.call(object, k)) {
+        if (fn(object[k], k, object) === false) {
+          return;
+        }
+      }
+    }
+  }
+};
+
+exports.events = (function (){
```

Comment:
Awesome! I think we can probably extend this to use for all the event handling in prebid.js if we tweak it a bit. I'll play with it a bit...